### ISSUE TITLE
Correct way for oom-killer alerts

### ISSUE THREAD

**Issue by cryptofuture** [firehol/netdata#2338]
What I already done:  
All my netdata hosts run with OOM score `-50` (could set in to -1000 possibly, through -50 enough to be alive for a longer time)  
I configured email notifications, so I could receive email alerts from netdata from every host.
I'm common with [alerts writing](https://github.com/firehol/netdata/wiki/health-configuration-examples)

Right know, I have doubts, do we have some netdata plugin that I could use for oom-killer kills monitoring or I need to write my own?
Basically what I want is an email notifications, that oom-killer killed **process name**.
I already upgraded ram on problem hosts, but I have no idea how long I could live with it, and when next "black friday" could  happen.

**Comment by ktsaou** [comment]
I think that netdata 1.8 (not the current github version, but the one after we release the current feature set) will have a central health monitoring service, that will detect if any of your systems has crashed, rebooted, frozen (or netdata killed) and send you a notification.

So, just a little bit more patience...

**Comment by ktsaou** [comment]
Cleaning up. If you need further help, just post here.
